# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/canvas/nsICanvasRenderingContextInternal.h
# Commit: 6d6c78c31c28
# Full Hash: 6d6c78c31c28c6d6827abfb392ccab10a24381ad
# Author: Andrew Osmond <aosmond@mozilla.com>
# Date: 2021-12-10 09:10:34
# Regressor Bug: 1736177
# File Overlap Count: 2
# Description:
#   Bug 1736177 - Part 8. Remove unused visibility observer from HTMLCanvasElement. r=jgilbert
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D130784
# ==============================================================================

diff -r d55f1ee88bb9 -r 6d6c78c31c28 dom/canvas/nsICanvasRenderingContextInternal.h
--- a/dom/canvas/nsICanvasRenderingContextInternal.h	Thu Dec 09 19:25:27 2021 +0000
+++ b/dom/canvas/nsICanvasRenderingContextInternal.h	Thu Dec 09 19:25:27 2021 +0000
@@ -183,8 +183,6 @@
     return nsString();
   }
 
-  virtual void OnVisibilityChange() {}
-
   virtual void OnMemoryPressure() {}
 
   virtual void OnBeforePaintTransaction() {}